src/scss/components/ec-card/ec-card.story.js

```diff
@@ -4,54 +4,63 @@ import EcIcon from '@/components/ec-icon';
 const stories = storiesOf('Card', module);
```

Comment:
Move this file to the `styles/components/ec-card` folder and delete the `scss/components/ec-card` folder.